tamr_toolbox/utils/version.py

```diff
@@ -49,8 +53,128 @@ def _as_float(version: str) -> float:
     return (version_split[0] * 1000) + version_split[1] + (version_split[2] / 10)
 
 
+def _get_tamr_versions_from_function_args(*args, **kwargs) -> List[str]:
+    all_args = locals()
+    args = [arg for arg in all_args["args"]]
+    kwargs = list(all_args["kwargs"].values())
+    all_args_parsed = args + kwargs
+    response = []
+
+    # Return the client version (if we can find it)
+    for arg in all_args_parsed:
+        if type(arg) is Client:
+            response.append(current(arg))
+        elif hasattr(arg, "client"):
+            if type(arg.client) is Client:
+                response.append(current(arg.client))
+
+    return response
+
+
+def is_version_condition_met(
```

Comment:
I'm unsure about this function name.  It isn't bad, but other options might be `check_tamr_version` or `validate_tamr_version`.

Reply:
I'm admittedly very attached to "is" being in the name because it returns a Boolean, and so the function name becomes a question, making it self descriptive.

Definitely agree it's awkward wording though, I'll have a think and DM you before I recommit

Reply:
I think that one implies that the function checks if the version string is a valid release of Tamr. (Which would be a good function let's be honest!)